FreeIT/src/lesson_12/additionalTask1/AdditionalTask1.java

```diff
@@ -0,0 +1,52 @@
+package lesson_12.additionalTask1;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
```

Comment:
some unused imports. You can select checkbox during commit to remove all unused imports